nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-services-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,327 @@
+nifi-cassandra-services-nar
+Copyright 2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) DataStax Java Driver for Apache Cassandra - Core
+      The following NOTICE information applies:
+        DataStax Java Driver for Apache Cassandra - Core
+        Copyright (C) 2012-2017 DataStax Inc.
+
+  (ASLv2) Apache Avro

```

Comment:
Should this be referenced here? The controller service never uses any Avro dependency.

Reply:
It can be removed.